compose/config/compose_spec.json

```diff
@@ -87,7 +87,7 @@
                 "dockerfile": {"type": "string"},
                 "args": {"$ref": "#/definitions/list_or_dict"},
                 "labels": {"$ref": "#/definitions/list_or_dict"},
-                "cache_from": {"$ref": "#/definitions/list_of_strings"},
+                "cache_from": {"type": "array", "items": {"type": "string"}},
```

Comment:
We download this file from the compose specification repo. Please propose this change there first: https://github.com/compose-spec/compose-spec/blob/68a799e866c2a8ff5d553c3797564282347d2659/schema/compose-spec.json#L90

Reply:
Thanks, the change has now been made upstream so this branch can be merged.